### ISSUE TITLE
Page title with different text size across ui_next

### ISSUE THREAD

**Issue by tiagodread** [ansible/awx#8465]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY

The dashboard and Management Jobs has a different text size <h2> while in other place usingn <h1>

##### ENVIRONMENT
* AWX version: 3.8.0
* AWX install method: docker for mac
* Ansible version:  2.9.13
* Operating System: Catalina
* Web Browser: Chrome

##### STEPS TO REPRODUCE

1. Navigate to dashboard or Management jobs

##### EXPECTED RESULTS

Page title should display the title using the same pattern for all places

##### ACTUAL RESULTS

Different tags in some pages

##### ADDITIONAL INFORMATION

![image](https://user-images.githubusercontent.com/12464858/97308146-4be16100-183f-11eb-9785-dc2938026d97.png)

![image](https://user-images.githubusercontent.com/12464858/97308170-53086f00-183f-11eb-97f5-d6d2b9ee1096.png)

**Comment by unlikelyzero** [comment]
@tiagodread does this violate an a11y standard?

**Comment by tiagodread** [comment]
> @tiagodread does this violate an a11y standard?

Not necessarily but we need keep the consistency between the importance of that information for the user (reading or listening)

**Comment by tiagodread** [comment]
Closing this after merge #8482

**Comment by jakemcdermott** [comment]
reopening - in the above screenshots, we actually want the font size shown in the dashboard image for all of the title bars.

**Comment by jlmitch5** [comment]
This is what we want it to look like:

https://www.patternfly.org/v4/demos/primary-detail/react-demos/primary-detail-content-padding/

So for example

```
Organizations > Ansible (two breadcrum links)
Details (on no line, title component)
```

**Comment by mabashian** [comment]
I believe that @jlmitch5 addressed this with #9083

**Comment by tiagodread** [comment]
Yeah, we can close this issue.
